go.mod

```diff
@@ -1,13 +1,20 @@
 module github.com/bitrise-steplib/steps-activate-ssh-key
 
-go 1.16
+go 1.18
```

Comment:
Since this is a critical step, I'd rather choose a fairly old Go version that's supported even by our frozen and deprecated stacks.